cfgov/unprocessed/css/on-demand/job_listing_list.less

```diff
@@ -14,3 +14,12 @@
     font-weight: normal;
   }
 }
+
+//adjust white space in job listing area on mobile
+.respond-to-max(@bp-xs-max, {
+  .o-sidebar-breakout_col {
+    .block.block__flush-top {
+      margin-bottom: 0
+    }
```

Comment:
Utility classes should not have their styles redefined (so no adding of styles to `block` or `block__flush-top`). Can this be applied directly to `o-sidebar-breakout_col`?

Reply:
margin-bottom is already being applied by "block__flush-bottom". Will look for another solution, thanks.